src/interfaces.ts

```diff
@@ -149,13 +151,13 @@ export interface LibraryListObject {
   label: string;
 }
 
-export interface ErrorResponse {
+export interface ProblemDetail {
```

Comment:
Ah, so now we have consistency between the Card Creator API, the internal API, and the front-end.